src/current/v23.1/migration-strategy-lift-and-shift.md

```diff
@@ -0,0 +1,142 @@
+- [Design a scalable process](#scalable)
+
+<a name="downtime"></a>
+
+### Decide how much downtime you can tolerate
+
+Depending on how much downtime you can tolerate, the following types of 'lift and shift' data migrations are possible:
+
+- If you can tolerate more downtime, a full backup from the source and restore into the target is the recommended approach.  For example, you could do one of the following:
+  - Export a SQL schema and CSV table data from the source database, and import them into CockroachDB using the [`IMPORT INTO`](import-into.html) statement.
+  - [Use a managed migration service](#managed-migration)
+- If the allowed downtime is too short for the backup/restore approach, you can use a logical replication tool such as [CDC sink](https://github.com/cockroachlabs/cdc-sink), and use the shorter downtime mainly to allow the pipeline to drain.
+
+<a name="managed-migration"></a>
+
+### Decide if you will use a managed migration service
+
+If you plan to do your bulk data migration using a managed migration service, you must have a secure, publicly available CockroachDB cluster. CockroachDB supports the following migration services:
+
+- [Migrate with AWS Database Migration Service (DMS)](aws-dms.html)
+- [Migrate and Replicate Data with Qlik Replicate](qlik.html)
+- [Migrate and Replicate Data with Striim](striim.html)
+
+{{site.data.alerts.callout_info}}
+Depending on the migration service you choose, [long-running transactions](query-behavior-troubleshooting.html#hanging-or-stuck-queries) can occur.  In some cases, these queries will cause [transaction retry errors](common-errors.html#restart-transaction). If you encounter these errors while migrating to CockroachDB using a managed migration service, please reach out to our [Support Resources](support-resources.html).
+{{site.data.alerts.end}}
+
+<a name="data-formats"></a>
+
+### Decide which data formats and storage media you will use
+
+It's important to decide which data formats, storage media, and database features you will use to migrate your data.
+
+Data formats that can be imported by CockroachDB include:
+
+- [SQL](schema-design-overview.html) for the [schema import](../cockroachcloud/migrations-page.html).
+- [CSV](migrate-from-csv.html) for table data.
+- [Avro](migrate-from-avro.html) for table data.
+
+The storage media you use to export / import from can be intermediate data files or streaming data coming over the network. Options include:
+
+- Local "userdata" storage (see [`cockroach userdata`](cockroach-userfile-upload.html), [Use a Local File Server](use-a-local-file-server.html)).
+- Cloud blob storage (see [`IMPORT INTO`](import-into.html), [Import Performance Best Practices](import-performance-best-practices.html), [Use Cloud Storage](use-cloud-storage.html)).
+- Direct wire transfers (see [github.com/cockroachlabs/cdc-sink](https://github.com/cockroachlabs/cdc-sink) or [managed migration services](#managed-migration)).
```

Comment:
@awoods187 I think we should avoid talking about CDC for migration use cases for now, right?

Reply:
For ingestion, it should be import/copy, aws dms, qlik, and striim, and i guess technically inserts/upserts. I think we should not include CDC sink until we decided separately.

Reply:
thanks, I've deleted all references to cdc-sink in this doc for now